index-handlers/src/test/java/no/sikt/nva/nvi/index/IndexDocumentHandlerTest.java

```diff
@@ -328,6 +327,29 @@ void shouldExtractOptionalPrintIssnFromExpandedResource(ChannelType channelType,
         assertEquals(expectedIndexDocument, actualIndexDocument);
     }
 
+    @Test
+    void shouldExtractContributorsPreviewAndContributorsCountFromExpandedResource() {
+        var candidate = randomApplicableCandidate(HARD_CODED_TOP_LEVEL_ORG, randomUri());
+        var expectedIndexDocument = setUpExistingResourceInS3AndGenerateExpectedDocument(candidate).indexDocument();
+        var event = createEvent(candidate.getIdentifier());
+        mockUriRetrieverOrgResponse(candidate);
+        handler.handleRequest(event, CONTEXT);
+        var actualIndexDocument = parseJson(s3Writer.getFile(createPath(candidate))).indexDocument();
+        assertEquals(expectedIndexDocument, actualIndexDocument);
+    }
+
+    @Test
+    void shouldSetContributorsPreviewToFiveFirstContributorsIfExpandedResourceIsMissingContributorsPreview() {
```

Comment:
Might incidentally want to test what happens when the magic node is an empty array, null, etc.